app/routes/github.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
+
+const {
+  get,
+  inject: { service },
+  Route
+} = Ember;
+
+const CODE_INVALID = 'The GitHub authorization code is invalid.';
+
+export default Route.extend(AuthenticatedRouteMixin, {
+  queryParams: { code: { refreshModel: true } },
```

Comment:
The `params` argument in the model hook is built using info provided in the `queryParams` hash defined in the route. It can alternatively be defined in the controller to, but either way, without `code` specified here, the first argument in the model hook will not contain a `code` key.

The second argument, which is a transition, will somewhere, deeply nested, contain a code key, but not the first argument.